### PR TITLE
use new invert() three.js v123 syntax

### PR DESCRIPTION
**Pull request by arpu**
will do some testing bevor ready

### CODE REVIEW

File: example/distancecast.js [hunk 1]
```diff
@@ -274,7 +274,7 @@ function updateDistanceCheck( fastCheck ) {
 	targetContainer.updateMatrixWorld();
 	const targetToBvh =
 		new THREE.Matrix4()
-			.getInverse( terrain.matrixWorld )
+		    .copy( terrain.matrixWorld ).invert()
```
Comment (gkjohnson) [inline example/distancecast.js:277]: Can we fix the indentation to be a tab here? I'll have fix the linting to make sure it runs on the example files, too 🤔

**Comment by gkjohnson** [comment]
Thanks! Very helpful. I was having some trouble building (perhaps a new node version or dependency versions broke) so I updated master to use the latest babel -- not sure if you were seeing similar issues.

File: example/distancecast.js (hunk 1 above)
Comment (arpu) [reply example/distancecast.js:277]: hm i add example to lint but lint does not show this

Comment (gkjohnson) [reply example/distancecast.js:277]: Thanks, I'll take a look!

**Pull request merged**